Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.6
+FROM python:3.10.4
```

Comment:
Could we swap to `python:3.10.4-slim` here? Should work as a straight swap, we've already checked it's fine on the slim image.